sparkhpc/sparkjob.py

```diff
@@ -158,10 +160,15 @@ def __init__(self,
         spark_home: 
             path to spark directory; default is the `SPARK_HOME` environment variable, 
             and if it is not set it defaults to `~/spark`
+        master_log_dir:
+            path to directory; default is {spark_home}/logs
+        master_log_filename:
```

Comment:
default value?

Reply:
Strange... it's in my repo on line 167 and in commit 9e850f67b3715143429b9e1ff1f4e7eeb9cb0e0c but it doesn't show up in the code review.

Reply:
ah indeed -- a glitch perhaps. Thanks!